src/components/App.js

```diff
@@ -1,20 +1,37 @@
 import React, { PropTypes } from 'react';
-import { Link, IndexLink } from 'react-router';
+import { Route } from 'react-router';
+import { Switch, NavLink } from 'react-router-dom';
+// 'Switch' & 'NavLink' imports have moved from the 'react-router' API to 'react-router-dom' since rrV4;
+// Also, IndexLink is replaced by NavLink with the activeStyle attr.
+// Switch is required for NotFoundPage to be the last case scenario.
+
+import HomePage from './HomePage';
+import FuelSavingsPage from '../containers/FuelSavingsPage'; // eslint-disable-line import/no-named-as-default
```

Comment:
consider removing this eslint-disable, your code meets the import/no-named-as-default rule afaict.